src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +64,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
+                + ", age=" + age + '}';
+    }
```

Comment:
and this may too

Reply:
Thank you for comment. Yes, maybe a little bit. That is addition method override i used to display of user data during debug process. i think its reasonable to use in our study process)) If you don't mind i will ask mentors in which cases we need use it.